iroh-relay/src/server/client.rs

```diff
@@ -557,6 +567,41 @@ impl Sink<Frame> for RateLimitedRelayedStream {
     }
 }
 
+/// Counts how many `NodeId`s seen, how many times.
+/// Gets reset every day.
+#[derive(Debug)]
+struct ClientCounter {
+    clients: HashMap<NodeId, usize>,
```

Comment:
So `usize` is how many times this has `NodeId` has been seen?  Is that a useful thing?  But also... you don't ever increase the value of it past `0`?  I'm a bit confused.